into/backends/json.py

```diff
@@ -192,4 +193,18 @@ def json_dumps(dt):
     return dt.isoformat()
 
 
+@convert.register(chunks(list), chunks(JSON))
+def convert_glob_of_jsons_into_chunks_of_lists(jsons, **kwargs):
+    def _():
+        return concat(convert(chunks(list), js, **kwargs) for js in jsons)
+    return chunks(list)(_)
+
+
+@convert.register(chunks(Iterator), chunks(JSONLines))
+def convert_glob_of_jsons_into_chunks_of_lists(jsons, **kwargs):
```

Comment:
Is there any reason why we don't have access to the first argument of the signature in `convert`? One reason I could see is that it prevents us from accidentally appending, but it also makes us have to write implementations that are the same code over and over again. If we had access to the type of the resulting conversion we could make a generic implementation that would do this. Just a thought.